velox/functions/prestosql/window/tests/LeadLagTest.cpp

```diff
@@ -58,6 +58,9 @@ TEST_P(LeadLagTest, offset) {
       makeFlatVector<int64_t>({1, 2, 3, 1, 2}),
       // Offsets with nulls.
       makeNullableFlatVector<int64_t>({1, 2, 3, std::nullopt, 2}),
+      // Large offsets(larger than uint32_t's max).
```

Comment:
nit: similar to other place, let's use a variable

```
int64_t largeOffset = <max int32_t> + 1;
```

Reply:
done